js/exporter/jspdf/v3/export_data_grid_3.js

```diff
@@ -103,10 +103,28 @@ function exportDataGrid(doc, dataGrid, options) {
                 x: doc.internal.pageSize.getWidth() - options.margin.right
             };
             const onSeparateRectHorizontally = (sourceRect, leftRect, rightRect) => {
-                const newRectCellInfo = Object.assign({}, sourceRect.sourceCellInfo, { text: '', debugSourceCellInfo: sourceRect.sourceCellInfo });
+                let leftRectTextOptions = {};
+                let rightRectTextOptions = {};
+                const isTextNotEmpty = isDefined(sourceRect.sourceCellInfo.text) && sourceRect.sourceCellInfo.text.length > 0;
+                if(isTextNotEmpty) {
+                    const isTextWidthGreaterThanRect = doc.getTextWidth(sourceRect.sourceCellInfo.text) > leftRect.w;
+                    const isHorizontalAlignSpecified = isDefined(sourceRect.sourceCellInfo.horizontalAlign) && sourceRect.sourceCellInfo.horizontalAlign !== 'left';
```

Comment:
sourceRect.sourceCellInfo.horizontalAlign !== 'left' - it means that NON-left value is specified but variable name is 'isHorizontalAlignSpecified' which means 'ANY value is specified'.

Reply:
Ok, i'l will change this variable name